cypress/plugins/index.js

```diff
@@ -24,4 +24,20 @@
 module.exports = (on, config) => {
   // `on` is used to hook into various events Cypress emits
   // `config` is the resolved Cypress config
+  on('before:browser:launch', (browser = {}, launchOptions) => {
```

Comment:
The same here, could you elaborate a bit? seems not related to this PR

Reply:
Sure, added comment above the code.